python/cudf/cudf/dataframe/numerical.py

```diff
@@ -500,3 +494,28 @@ def digitize(column, bins, right=False):
     bins_buf = Buffer(rmm.to_device(bins))
     bin_col = NumericalColumn(data=bins_buf, dtype=bins.dtype)
     return cpp_sort.digitize(column, bin_col, right)
+
+
+def is_zero_in_column(col=None):
```

Comment:
Instead of `is_zero_in_column` why don't we just make this `__contains__`: https://docs.python.org/3/reference/datamodel.html#object.__contains__ where you can pass an arbitrary scalar and it returns True or False.

Reply:
Then you can do `0 in my_column` and it will execute on the GPU instead of via Python iteration.